atmat/atmexall.m

```diff
@@ -5,6 +5,9 @@ function atmexall(varargin)
 % properly configured.
 % On Windows, Microsoft Visual C++ is required
 
+% Add AT directories to the Matlab path
+atpath
+
 PLATFORMOPTION = ['-D',computer,' ',sprintf('%s ',varargin{:})];
 LIBDL='';
 switch computer

```

Comment:
This results in a double definition of all the Matlab path ! It is useless if AT is correctly installed, and it is is not, I do not see how you call `atmexall`. More generally, I do not like modifications of the Matlab path without the user's intervention.